provisioner/k8s/controllers/bundle_controller.go

```diff
@@ -202,34 +202,24 @@ func (r *BundleReconciler) ensureUnpackPod(ctx context.Context, bundle *olmv1alp
 		pod.Spec.AutomountServiceAccountToken = &automountServiceAccountToken
 		pod.Spec.Volumes = []corev1.Volume{
 			{Name: "util", VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}}},
-			{Name: "bundle", VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}}},
 		}
 		pod.Spec.RestartPolicy = corev1.RestartPolicyNever
-		if len(pod.Spec.InitContainers) != 2 {
-			pod.Spec.InitContainers = make([]corev1.Container, 2)
+		if len(pod.Spec.InitContainers) != 1 {
+			pod.Spec.InitContainers = make([]corev1.Container, 1)
 		}
 		pod.Spec.InitContainers[0].Name = "install-cpb"
-		pod.Spec.InitContainers[0].Image = r.CopyBundleImage
-		pod.Spec.InitContainers[0].Command = []string{"/bin/cp", "-Rv", "/bin/cpb", "/util/cpb"}
+		pod.Spec.InitContainers[0].Image = r.UnpackImage
+		pod.Spec.InitContainers[0].Command = []string{"cp", "-Rv", "/unpack", "/util/unpack"}
 		pod.Spec.InitContainers[0].VolumeMounts = []corev1.VolumeMount{{Name: "util", MountPath: "/util"}}
 
-		pod.Spec.InitContainers[1].Name = "copy-bundle"
-		pod.Spec.InitContainers[1].Image = bundle.Spec.Image
-		pod.Spec.InitContainers[1].ImagePullPolicy = corev1.PullAlways
-		pod.Spec.InitContainers[1].Command = []string{"/util/cpb", "--ignore-annotations-file", "/bundle"}
-		pod.Spec.InitContainers[1].VolumeMounts = []corev1.VolumeMount{
-			{Name: "util", MountPath: "/util"},
-			{Name: "bundle", MountPath: "/bundle"},
-		}
-
 		if len(pod.Spec.Containers) != 1 {
 			pod.Spec.Containers = make([]corev1.Container, 1)
 		}
-		pod.Spec.Containers[0].Name = "unpack-bundle"
-		pod.Spec.Containers[0].Image = r.UnpackImage
+		pod.Spec.Containers[0].Name = "copy-bundle"
```

Comment:
Nit: maybe just `bundle`?
```suggestion
		pod.Spec.Containers[0].Name = "bundle"
```

Reply:
That's fine with me - we parse the name of the copy-bundle container to get the image digest of the bundle.Spec.Image, so I created a constant to track the name of the container that's responsible for unpacking bundle contents to stdout.